README.md

```diff
@@ -147,12 +147,77 @@ client
     template_variables: {
       user_name: "John Doe",
     },
-    subject: "Hello from Mailtrap!"
   })
   .then(console.log)
   .catch(console.error);
 ```
 
+### Nodemailer Transport
+
+You can provide Mailtrap specific keys like `category`, `customVariables`, `templateUuid` and `templateVariables`.
+
+```ts
+import { readFileSync } from "fs";
+import Nodemailer from "nodemailer";
```

Comment:
We should make it clear that users would need to install `nodemailer` by themselves

Reply:
@VladimirTaytor Will mention that in README.md file